common/src/python/enrollment/enrollment_subject.py

```diff
@@ -96,17 +97,18 @@ def update_enrollment_info(self, record: EnrollmentRecord) -> None:
     def upload_enrollment(self, record: EnrollmentRecord) -> None:
         """Uploads a file for the enrollment record to this subject.
 
-        The record is saved as a JSON file
-        `enrollment_transfer/enrollment/enrollment.json`
-        under this subject.
+        The record is saved as a JSON file under this subject.
 
         Args:
           record: the enrollment record
         """
+        formdate = record.start_date.strftime("%Y-%m-%d")
+        session_label = f'{DefaultValues.ENRL_SESSION_LBL_PRFX}{formdate}'
         self.upload_acquisition_file(
-            session_label='enrollment_transfer',
-            acquisition_label='enrollment',
-            filename='enrollment.json',
+            session_label=session_label,
+            acquisition_label=DefaultValues.ENROLLMENT_MODULE,
+            filename=
+            f'{record.naccid}_{session_label}_{DefaultValues.ENROLLMENT_MODULE}.json',
```

Comment:
if we are going to use this naming scheme as a convention, probably should make a function to create the name